app/javascript/Search/__tests__/Search.test.jsx

```diff
@@ -9,6 +9,7 @@ describe('<Search />', () => {
     global.filterXSS = jest.fn();
     global.InstantClick = jest.fn(() => ({
       on: jest.fn(),
+      off: jest.fn(),
```

Comment:
This was missing and never caused issues until I discovered we were using the legacy `componentDidUnmount`.